netmiko/dnos/dnos_ssh.py

```diff
@@ -187,11 +187,15 @@ def find_prompt(
         else:
             return prompt
 
-    def _enter_shell(self) -> str:
-        """Enter the Bourne Shell of the routing-engine container on the
-        active NCC"""
+    def _enter_shell(self,ncm=None, ncc:int=0) -> str:
```

Comment:
Why are you defaulting ncc to 0, the active keyword was a conscious choice as the user may not know in advance which ncc is active before connecting to the box.